### PR TITLE
#08 - Wishlist and CartScreen -UI 

### PR DESCRIPTION
**Pull request by Koushith**
## Description

- [x] Adds My cart and Wishlist page- UI only.

## Screenshot
screenshot (2)
screenshot (4)

### CODE REVIEW

File: src/components/common/navbar.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const NabBar = () => {
```
Comment (sakshi006) [inline src/components/common/navbar.js:1]: Navbar*

File: src/components/discount-cards/discount.module.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.discount_container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 3rem;
+  gap: 2rem;
+  background: #ffd05a;
```
Comment (sakshi006) [inline src/components/discount-cards/discount.module.css:8]: Use CSS variables for color

File: src/components/discount-cards/discount.module.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  color: #d07b2d;
+  max-width: 46.9rem;
+  border-radius: 0.5rem;
```
Comment (sakshi006) [inline src/components/discount-cards/discount.module.css:11]: border radios shoud be in px as we don't want it to change with screen sizes

File: src/components/product-card/product-card.component.js [hunk 4]
```diff
@@ -1,10 +1,58 @@
...
+    // <div class='card ecomm'>
+    //   <div class='product-image'>
+    //     <img
+    //       src='https://assets.myntassets.com/h_1440,q_90,w_1080/v1/assets/images/productimage/2021/2/18/07b7a314-decd-421a-b447-01b2411e24e61613643428870-1.jpg'
+    //       alt='card-pic'
+    //     />
+    //   </div>
+    //   <div class='card-badge'>NEW</div>
+    //   <h2 class='card-title'>Athena top</h2>
+    //   <div class='card-text'>4 star</div>
```
Comment (sakshi006) [inline src/components/product-card/product-card.component.js:16]: Remove commented code

**Review by sakshi006 — APPROVED** [review]
The code structure and naming conventions are good. Made some comments, please look into it

File: src/components/common/navbar.js (hunk 1 above)
Comment (Koushith) [reply src/components/common/navbar.js:1]: yeah, my bad. its a typo.

File: src/components/discount-cards/discount.module.css (hunk 2 above)
Comment (Koushith) [reply src/components/discount-cards/discount.module.css:8]: will update it in the component lib. haven't finalized the colors yet.

File: src/components/product-card/product-card.component.js (hunk 4 above)
Comment (Koushith) [reply src/components/product-card/product-card.component.js:16]: done.

**Pull request merged**